src/libs/episodes-repo.ts

```diff
@@ -25,6 +25,10 @@ export function getEpisodeByNumber(episodeNumber: string): EpisodeData {
   return episodeMapByNumber.get(episodeNumber);
 }
 
+export function getRandomEpisode(): EpisodeData {
+  return episodeMapByNumber.get(`${Math.floor(Math.random() * (episodeMapByNumber.size)) + 1}`);
```

Comment:
Suggest using the array `episodeDatas`  instead, to not have to do the string converting nor the +1

```suggestion
  return episodeDatas[Math.floor(Math.random() * episodeDatas.length];
```

Though I think I had a utility function for this. Check the `libs/*.ts`, I think one of them has a function for it. The use case is really simple, but using an abstracted function improves readability